CreeDictionary/CreeDictionary/paradigm/filler.py

```diff
@@ -0,0 +1,519 @@
+from __future__ import annotations
+
+"""
+Fill a paradigm table with the inflections of a lemma.
+"""
+
+import logging
+from copy import deepcopy
+from pathlib import Path
+from string import Template
+from typing import Iterable, Literal, Optional, Sequence, Union, cast
+
+from attr import attrib, attrs
+from hfst_optimized_lookup import TransducerFile
+from utils import ParadigmSize, WordClass, shared_res_dir
+from utils.types import ConcatAnalysis
+
+logger = logging.getLogger()
+
+PARADIGM_NAME_TO_WC = {
+    "noun-na": WordClass.NA,
+    "noun-nad": WordClass.NAD,
+    "noun-ni": WordClass.NI,
+    "noun-nid": WordClass.NID,
+    "verb-ai": WordClass.VAI,
+    "verb-ii": WordClass.VII,
+    "verb-ta": WordClass.VTA,
+    "verb-ti": WordClass.VTI,
+}
+
+FREQUENCY_FILENAME = "attested-wordforms.txt"
```

Comment:
@aarppe was telling me about this file the other day, and I believe that both the file and variable name could be renamed to something like `corpus_frequency`. The related `W_aggr_corp_morph_log_freq.txt` file synthesizes a rank for wordforms that do not occur in the corpus, using corpus morpheme frequencies; the details on that are in #163.